packages/backend/src/projectAdapters/projectAdapter.ts

```diff
@@ -25,11 +25,15 @@ export const projectAdapterFromConfig = async (
         warehouseClientFromCredentials(warehouseCredentials);
     const configType = config.type;
     Logger.debug(`Initialize project adaptor of type ${configType}`);
+
     switch (config.type) {
         case DbtProjectType.DBT:
             return new DbtLocalCredentialsProjectAdapter({
                 warehouseClient,
-                projectDir: config.project_dir || '/usr/app/dbt',
+                projectDir:
+                    config.project_dir ||
+                    process.env.DBT_PROJECT_DIR ||
```

Comment:
This change doesn't seem right since the app shouldn't use the env `DBT_PROJECT_DIR`. this env is only useful when setting up the docker-compose to create a volume.

Reply:
@ZeRego 
then im not sure what's this instruction is for. config.project_dir is empty btw and it fallbacks to `/usr/app/dbt` which works for Docker but user will not be able to override it.

![CleanShot 2022-10-07 at 17 10 40@2x](https://user-images.githubusercontent.com/962095/194561646-2e3fb5d8-7442-4eaa-a5e5-856be5523626.png)

Reply:
Docker compose creates a volume ( mapping between the user path to the dbt dir and /usr/app/dbt) 

```
    volumes:
      - "${DBT_PROJECT_DIR}:/usr/app/dbt"
```

>  config.project_dir is empty btw and it fallbacks to /usr/app/dbt

This is expected. As the path `/usr/app/dbt` in the docker container will map to the path they provided in the install script or via the DBT_PROJECT_DIR

Reply:
@IrakliJani not sure why do we need it, is it because you want to connect to local projects when developing? let's move this conversation to slack :)